aiida_quantumespresso/workflows/pdos.py

```diff
@@ -219,14 +222,15 @@ def define(cls, spec):
             help='Terminate workchain steps before submitting calculations (test purposes only).'
         )
         spec.input(
-            'align_to_fermi',
-            valid_type=orm.Bool,
+            'fermi_energy_range',
+            valid_type=orm.List,
```

Comment:
This also needs a validator to check that when specified, it contains two float values

Reply:
I added a validator. I split it into two parts. Maybe one would prefer to combine it by an `or` and return just one messsage.

Reply:
Validator looks good, just a suggestion on the error message to make it a bit clearer and more consistent